site/app/views/grading/ElectronicGraderView.php

```diff
@@ -958,6 +958,9 @@ public function hwGradingPage(Gradeable $gradeable, GradedGradeable $graded_grad
         $isDiscussionPanel = false;
         $isGradeInquiryPanel = false;
         $is_peer_grader = false;
+        // Converting array from '{"a", "b"}' to [a, b]
+        $isPeerAccessible = explode(",", trim($gradeable->getPeerPanel(), '{}'));
```

Comment:
Why is $gradeable->getPeerPanel() the first parameter to trim() given it returns an array of integers?

Reply:
Thanks for the review,
Here's why `$gradeable->getPeerPanel()` is being trimmed : 
PostgreSQL supports array data types and lets us to store arrays in this format : "{"A", "B"}"
This is why im trimming this down to [A, B] 
Let me know if you want me to elaborate the comment in line 961

Also, I don't think its intended for limited-access grader accounts as per the issue # 10114